### ISSUE TITLE
Error with Fetch returning 204 No Content response

### ISSUE THREAD

**Issue by Izdatyi** [syumai/workers#122]
Hi!
Testing with [fetch](https://github.com/syumai/workers/blob/main/_examples/fetch/main.go) example
Just change for example: ([reqres.in](https://reqres.in/api-docs/#/default/delete_users__id_))

```go
r, err := fetch.NewRequest(req.Context(), http.MethodDelete, "https://reqres.in/api/users/1", nil)
```

Get an error
```[wrangler:inf] Ready on http://127.0.0.1:8787
panic: syscall/js: call of Value.Call on null
X [ERROR] A hanging Promise was canceled. This happens when the worker runtime is waiting for a Promise from JavaScript to resolve, but has detected that the Promise cannot possibly ever resolve because all code and events related to the Promise's I/O context have already finished.

[wrangler:inf] GET / 500 Internal Server Error (252ms)                                    
X [ERROR] Uncaught (in response) Error: The script will never generate a response.
```
Also checked with the GitHub API

**Comment by syumai** [comment]
I'll try to fix this, thanks!

**Comment by syumai** [comment]
@Izdatyi This issue was probably fixed in #124, so I've released it as [v0.26.3](https://github.com/syumai/workers/releases/tag/v0.26.3).
Please try this later.

**Comment by Izdatyi** [comment]
Works as expected.
Thank you for the quick fix!
